spec/player_factory_spec.rb

```diff
@@ -3,20 +3,22 @@
 require_relative '../lib/computer_player.rb'
 require_relative '../lib/console_user_interface.rb'
 require_relative '../lib/minimax.rb'
+require_relative '../lib/player_factory_validations.rb'
 
 describe PlayerFactory do
 
   let(:user_interface_double) { instance_double("ConsoleUserInterface") }
   let(:human_player_model) { HumanPlayer }
   let(:computer_player_model) { ComputerPlayer }
   let(:ai_double) { instance_double("Minimax") }
+  let(:validator_double) { instance_double("PlayerFactoryValidations") }
 
   let(:player_factory) { PlayerFactory.new }
   
   describe "#set_up" do 
-    it "requires keyword arguments for user_interface, human_player_model, computer_player_model, and ai" do 
+    it "requires keyword arguments for user_interface, human_player_model, computer_player_model, ai, and validator" do 
```

Comment:
These sorts of tests can be fragile and require updates every time you change an argument. If this is only an error message for a developer, I could see removing tests like this.